instances.go

```diff
@@ -201,6 +234,55 @@ func (c *Client) SetAllowedNetworkInstancePolicy(ctx context.Context, enable boo
 	return err
 }
 
+// SetKeyAccessInstancePolicy updates the key create import access policy details associated with an instance.
+// For more information can refer to the Key Protect docs in the link below:
+// <kp link>
+func (c *Client) SetKeyAccessInstancePolicy(ctx context.Context, enable bool, attributes map[string]bool) error {
+	policy := InstancePolicy{
+		PolicyType: KeyAccess,
+		PolicyData: PolicyData{
+			Enabled: &enable,
+		},
+	}
+
+	if enable {
+		policy.PolicyData.Attributes = &Attributes{}
+		a := policy.PolicyData.Attributes
+		a.CreateRootKey = validateValue(attributes, CreateRootKey)
+		a.CreateStandardKey = validateValue(attributes, CreateStandardKey)
+		a.ImportRootKey = validateValue(attributes, ImportRootKey)
+		a.ImportStandardKey = validateValue(attributes, ImportStandardKey)
+		a.EnforceToken = validateValue(attributes, EnforceToken)
+	}
+
+	policyRequest := InstancePolicies{
+		Metadata: PoliciesMetadata{
+			CollectionType:   policyType,
+			NumberOfPolicies: 1,
+		},
+		Policies: []InstancePolicy{policy},
+	}
+
+	err := c.setInstancePolicy(ctx, KeyAccess, policyRequest)
+	if err != nil {
+		return err
+	}
+
+	return err
+}
+
+func validateValue(attributes map[string]bool, key string) *bool {
+	True := true
+	False := false
```

Comment:
nitpick, it seems strange (to me) to export these 
```suggestion
	trueBool := true
	falseBool := false
```